lib/signaling/v2/room.js

```diff
@@ -53,6 +53,9 @@ class RoomV2 extends RoomSignaling {
         value: null,
         writable: true
       },
+      _networkQualityConfiguration: {
+        value: localParticipant._networkQualityConfiguration
```

Comment:
@syerrapragada If this is how you access the `NetworkQualityConfiguration`, then can you make it a public member of `LocalParticipantV2`?